### PR TITLE
fix:[CORE-1175]Merge vaults into one target type and remove ops alerts 

### PR DESCRIPTION
**Pull request by AishwaryaKulkarni97**
# Description

- Merge vaults with legacy and RBAC model into one
- Fix null pointer for security alerts
- remove duplicate policies for RBAC

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Chore (no code changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also
list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My commit message/PR follows the contribution guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# **Other information**:

List any documentation updates that are needed for the Wiki

### CODE REVIEW

**Review by raviteja-mandala — APPROVED** [review]

File: jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/collector/SecurityAlertsInventoryCollector.java
Comment (mkinchin-paladin) [inline jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/collector/SecurityAlertsInventoryCollector.java:53]: Update according to code style in this file.

File: jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/collector/VaultInventoryCollector.java
Comment (mkinchin-paladin) [inline jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/collector/VaultInventoryCollector.java:103]: Space after `if`

File: jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/collector/SecurityAlertsInventoryCollector.java
Comment (mkinchin-paladin) [reply jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/collector/SecurityAlertsInventoryCollector.java:53]: you can use autoformat if you are using InteliJ

File: jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/collector/SecurityAlertsInventoryCollector.java
Comment (mkinchin-paladin) [inline jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/collector/SecurityAlertsInventoryCollector.java:51]: I'd like to have this line up top, around line 46, so you can use this value for `setId` method and for region as well.
And it should look like this
```
String id = databricksObject.get("id").getAsString();
```

**Review by mkinchin-paladin — DISMISSED** [review]
Minor changes requested. You can push as soon as you update them, don't need wait for approval from me.

File: jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/collector/SecurityAlertsInventoryCollector.java
Comment (AishwaryaKulkarni97) [reply jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/collector/SecurityAlertsInventoryCollector.java]: Resolved in commit

File: jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/collector/SecurityAlertsInventoryCollector.java
Comment (AishwaryaKulkarni97) [reply jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/collector/SecurityAlertsInventoryCollector.java]: Resolved in commit

**Review by raviteja-mandala — APPROVED** [review]

**Pull request merged**
